fabric-transitive-access-wideners-v1/template.accesswidener

```diff
@@ -24,4 +24,104 @@ transitive-accessible field net/minecraft/loot/LootPool bonusRolls Lnet/minecraf
+
+# Registering custom scoreboard criteria
+transitive-accessible method net/minecraft/scoreboard/ScoreboardCriterion create (Ljava/lang/String;ZLnet/minecraft/scoreboard/ScoreboardCriterion$RenderType;)Lnet/minecraft/scoreboard/ScoreboardCriterion;
+transitive-accessible method net/minecraft/scoreboard/ScoreboardCriterion create (Ljava/lang/String;)Lnet/minecraft/scoreboard/ScoreboardCriterion;
+
+# Model part subclasses
+transitive-accessible class net/minecraft/client/model/ModelPart$Vertex
+transitive-accessible class net/minecraft/client/model/ModelPart$Quad
+
+# Creating custom render layers
+transitive-accessible method net/minecraft/client/render/RenderLayer of (Lnet/minecraft/client/render/RenderPhase$Shader;)Lnet/minecraft/client/render/RenderLayer$MultiPhaseParameters;
+
+# Creating custom sensor types
+transitive-accessible method net/minecraft/entity/ai/brain/sensor/SensorType <init> (Ljava/util/function/Supplier;)V
+
+# Creating custom activities
+transitive-accessible method net/minecraft/entity/ai/brain/Activity <init> (Ljava/lang/String;)V
+
+# Living entity methods
+transitive-accessible method net/minecraft/entity/LivingEntity damageArmor (Lnet/minecraft/entity/damage/DamageSource;F)V
+transitive-accessible method net/minecraft/entity/LivingEntity damageHelmet (Lnet/minecraft/entity/damage/DamageSource;F)V
+transitive-accessible method net/minecraft/entity/LivingEntity damageShield (F)V
+
+# Armor stand methods
+transitive-accessible method net/minecraft/entity/decoration/ArmorStandEntity isSlotDisabled (Lnet/minecraft/entity/EquipmentSlot;)Z
+transitive-accessible method net/minecraft/entity/decoration/ArmorStandEntity equip (Lnet/minecraft/entity/player/PlayerEntity;Lnet/minecraft/entity/EquipmentSlot;Lnet/minecraft/item/ItemStack;Lnet/minecraft/util/Hand;)Z
+transitive-accessible method net/minecraft/entity/decoration/ArmorStandEntity setSmall (Z)V
+transitive-accessible method net/minecraft/entity/decoration/ArmorStandEntity setShowArms (Z)V
+transitive-accessible method net/minecraft/entity/decoration/ArmorStandEntity setHideBasePlate (Z)V
+transitive-accessible method net/minecraft/entity/decoration/ArmorStandEntity setMarker (Z)V
+
+# Miscellaneous entity methods and constructors
+transitive-accessible method net/minecraft/entity/decoration/ItemFrameEntity getAsItemStack ()Lnet/minecraft/item/ItemStack;
+transitive-accessible method net/minecraft/entity/projectile/ProjectileEntity <init> (Lnet/minecraft/entity/EntityType;Lnet/minecraft/world/World;)V
+transitive-accessible method net/minecraft/entity/projectile/PersistentProjectileEntity asItemStack ()Lnet/minecraft/item/ItemStack;
+transitive-accessible method net/minecraft/entity/projectile/FishingBobberEntity <init> (Lnet/minecraft/entity/EntityType;Lnet/minecraft/world/World;II)V
+transitive-accessible method net/minecraft/entity/vehicle/AbstractMinecartEntity getItem ()Lnet/minecraft/item/Item;
```

Comment:
Why?

Reply:
Decided to remove the `getAsItemStack`, `asItemStack`, and `getItem` methods again. Kept the two constructors.